azure-pipelines.yml

```diff
@@ -22,6 +22,14 @@ trigger:
 pr: # https://docs.microsoft.com/en-us/azure/devops/pipelines/yaml-schema?view=azure-devops&tabs=schema%2Cparameter-schema#pr-trigger
   drafts: false
 
+# https://docs.microsoft.com/ru-ru/azure/devops/pipelines/process/scheduled-triggers?view=azure-devops&tabs=yaml#scheduled-triggers
```

Comment:
Could you please ref what gets executed as comment here ? :) It's [this one](https://github.com/KomodoPlatform/atomicDEX-API/blob/dev/azure-pipelines-build-stage-job.yml#L20) if I am not mistaken.

Reply:
@ozkanonur Done, thanks for the note 🙂